src/snowflake/snowpark/_internal/ast.py

```diff
@@ -25,17 +28,48 @@ def expr_to_dataframe_expr(expr):
     return dfe
 
 
+# Map from python type to its corresponding IR field. IR fields below all have the 'v' attribute.
```

Comment:
It's technically an "IR entity". These look like fields in the protobuf builder.